docker/Dockerfile.gateway

```diff
@@ -0,0 +1,24 @@
+
+ARG spdk_version
+
+FROM spdk:${spdk_version:-latest} as spdk
+
+#FROM fedora:33 AS base
```

Comment:
Let's we use UBI (https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image) as the base image ?

Reply:
We had a discussion with Ilya re the container. There is a ceph project that builds ceph related containers
This was a quick and easy method to build a gateway container that can be used for testing.